sources/packages/api/src/services/application/application.service.ts

```diff
@@ -503,15 +503,22 @@ export class ApplicationService extends RecordDataModelService<Application> {
         "offering.studyStartDate",
         "offering.studyEndDate",
         "msfaaNumber.msfaaNumber",
+        "disbursementSchedule.disbursementDate",
+        "disbursementValue.valueType",
+        "disbursementValue.valueCode",
+        "disbursementValue.valueAmount",
       ])
       .innerJoin("application.student", "student")
       .innerJoin("student.user", "user")
       .innerJoin("application.offering", "offering")
       .innerJoin("offering.educationProgram", "educationProgram")
       .innerJoin("application.location", "location")
       .innerJoin("application.msfaaNumber", "msfaaNumber")
+      .innerJoin("application.disbursementSchedules", "disbursementSchedule")
+      .innerJoin("disbursementSchedule.disbursementValues", "disbursementValue")
       .where("application.id = :applicationId", { applicationId })
       .andWhere("student.id = :studentId", { studentId })
+      .addOrderBy("disbursementSchedule.disbursementDate")
```

Comment:
Please replace the "addOrderBy" by "orderBy". In my understanding, the "addOrderBy" is supposed to be used when you already have an "orderBy" in place.

Reply:
Review comment addressed.